modules/message-broker-impl/src/main/java/org/opencastproject/message/broker/impl/MessageReceiverImpl.java

```diff
@@ -46,12 +46,12 @@
  * A class to receive messages from a ActiveMQ Message Broker.
  */
 @Component(
-  property = {
+    property = {
     "service.description=Message Broker Receiver",
     "service.pid=org.opencastproject.message.broker.impl.MessageReceiverImpl"
```

Comment:
Shouldn't this be indented one more level?
There are a few identical cases.

Reply:
Absolutely. So I guess I really need to pay closer attention to that and not just let my script indent everything according to how checkstyle wants it.

Unfortunately, checkstyle is still buggy when it comes to indenting these annotations. And it often suggests absolutely wrong values. So these things are not caught automatically. But yeah, gonna pay more manual attention to that from now on. And the cases in this PR should be fixed.